internal/pkg/build/build.go

```diff
@@ -47,6 +48,9 @@ type Build struct {
 
 // Config defines how build is executed, including things like where final image is written.
 type Config struct {
+	// Image cache handle
+	ImgCache *cache.Handle
```

Comment:
I would consider including this in the options struct in https://github.com/sylabs/singularity/blob/master/pkg/build/types/bundle.go#L43. This way you can access the handle without needing to expand the conveyorPacker interface and without needing to set dummy functions for converyorPackers that dont use the image cache. Any conveyorPacker that needs it can simply get the handle from the bundle in their `Get()` method.